_posts/2016-02-29-twis-53.md

```diff
@@ -0,0 +1,36 @@
+---
+layout:     post
+title:      This Week In Servo 53
+date:       2016-02-29 00:30:00
+summary:    What's up with Servo for the week of 22 Feb 2016 - 29 Feb 2016
+categories:
+---
+
+In the [last week](https://github.com/pulls?page=1&q=is%3Apr+is%3Amerged+closed%3A2016-02-22..2016-02-29+user%3Aservo), we landed 115 PRs in the Servo organization's repositories.
+
+Josh Matthews truly delivered for Servo this week! He went through all of our stale PRs, rebased, reviewed, or closed them, and took us from around 70 open PRs down to around 45! Great work, Josh.
+
+We're very excited to announce that two long-time Servo community members and reviewers will be joining the Servo team full time! Anthony Ramine (nox) will be starting on March 14th, working out of the Paris office (alongside Simon Sapin). Manish Goregaokar (manishearth) will be starting officially on May 23rd, and will relocate to the SF office alongside Patrick Walton in August.
```

Comment:
Probably should mention that one of the positions is still open to avoid confusion; folks keep asking about that since they haven't heard.

Also, September :smile: